src/React/Resources/assets/package.json

```diff
@@ -0,0 +1,31 @@
+{
+    "name": "@symfony/ux-react",
+    "description": "Integration of React in Symfony",
+    "license": "MIT",
+    "version": "1.0.0",
+    "main": "dist/register_controller.js",
+    "symfony": {
+        "controllers": {
+            "react": {
+                "main": "dist/render_controller.js",
+                "webpackMode": "eager",
+                "fetch": "eager",
+                "enabled": true
+            }
+        }
+    },
+    "peerDependencies": {
+        "@hotwired/stimulus": "^3.0.0",
+        "react": "^16.0 || ^17.0 || ^18.0",
+        "react-dom": "^16.0 || ^17.0 || ^18.0"
+    },
+    "dependencies": {
+        "@types/webpack-env": "^1.16"
```

Comment:
Is this needed? I assume it's for some Typescript types? Is it needed in `dependencies` or is it a `devDependency`?

Reply:
Yes, it is used by registerReactControllerComponents and I think Typescript users of the package would need to have it installed for it to work (or perhaps is Typescript not checking the vendors? would make sense)